doc/src/errorrecovery.md

```diff
@@ -537,11 +537,15 @@ the first parsing error, with the `recoverer` method in `CTParserBuilder` or
 `RTParserBuilder`. For example, we can change `calc`'s `build.rs` file to:
 
 ```rust,noplaypen
-    let lex_rule_ids_map = CTParserBuilder::new()
-        .yacckind(YaccKind::Grmtools)
-        .recoverer(lrpar::RecoveryKind::None)
-        .grammar_path_in_src("calc.y")?
-        .process()?;
+CTLexerBuilder:::new()
```

Comment:
Oops,  ':::new()', here should be '::', twas my bad.

Reply:
My bad! Fixed in 8901c94.